llama_stack/providers/tests/inference/test_text_inference.py

```diff
@@ -171,7 +171,7 @@ async def test_chat_completion_non_streaming(
     ):
         inference_impl, _ = inference_stack
         response = await inference_impl.chat_completion(
-            model=inference_model,
+            model_id=inference_model,
```

Comment:
https://github.com/meta-llama/llama-stack/blob/e51107e019771909b8154b2f024448bb41c45870/llama_stack/providers/inline/eval/meta_reference/eval.py#L152-L154

/eval also calls chat_completion, which also need to be changed?

Reply:
done.